### PR TITLE
Different clocks and mute option

### PR DESCRIPTION
**Pull request by bblaszkow06**

### CODE REVIEW

File: lib/timer/libshout.ex [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+  @impl Timer
+  def current_time() do
+    native_time() |> Time.milliseconds()
+  end
+
+  @impl GenServer
+  def init([target, interval, delay]) do
+    with interval = interval |> Time.to_milliseconds(),
+         delay = delay |> Time.to_milliseconds(),
```
Comment (mat-hek) [inline lib/timer/libshout.ex]: move outside of with

File: lib/mixer.ex [hunk 2]
```diff
@@ -222,42 +315,32 @@ defmodule Membrane.Element.LiveAudioMixer do
 
   defp update_outputs(outputs, skip_add) do
```
[the remark this answers is not in the document]
Comment (bblaszkow06) [reply lib/mixer.ex]: How? Maybe `skip_add` could be renamed to `skip_to_add`, but I don't see much room for improvement.

[the remark this answers is not in the document]
Comment (mat-hek) [reply lib/mixer.ex]: `defp update_outputs_skips(outputs, expected_queue_size)` ?

[the remark this answers is not in the document]
Comment (bblaszkow06) [reply lib/mixer.ex]: This hides the most important thing about this function - that queues are flushed. You can check out my new proposal

**Pull request merged**